src/main/java/net/neoforged/neoforge/client/gui/widget/ModListWidget.java

```diff
@@ -86,11 +86,26 @@ public void render(GuiGraphics guiGraphics, int entryIdx, int top, int left, int
 
         @Override
         public boolean mouseClicked(double p_mouseClicked_1_, double p_mouseClicked_3_, int p_mouseClicked_5_) {
-            parent.setSelected(this);
-            ModListWidget.this.setSelected(this);
+            // clicking on a selected item a second time unselects it
+            parent.setSelected(isFocused() ? null : this);
+            ModListWidget.this.setSelected(isFocused() ? null : this);
             return false;
         }
 
+        @Override
+        public void setFocused(boolean p_265302_) {
+            // ignore focus loss so the item stays selected when tabbing to the config button
+            if (p_265302_) {
```

Comment:
```suggestion
        public void setFocused(boolean focused) {
            // ignore focus loss so the item stays selected when tabbing to the config button
            if (focused) {
```

Reply:
I hate guessing names. But here, where it's actually used, it makes sense.